### DISCUSSION TITLE
compose doesnt work

### DISCUSSION THREAD

**Discussion by rightsaidfred99** [nextcloud/all-in-one#2618] [Questions] [answered]
ports:
      - 8080:8080

     
Docker socket is not available. Cannot continue.
If you did this by purpose because you don't want the container to have access to the docker socket, see https://github.com/nextcloud/all-in-one/tree/main/manual-install.
Docker socket is not available. Cannot continue.
If you did this by purpose because you don't want the container to have access to the docker socket, see https://github.com/nextcloud/all-in-one/tree/main/manual-install.
Docker socket is not available. Cannot continue.
If you did this by purpose because you don't want the container to have access to the docker socket, see https://github.com/nextcloud/all-in-one/tree/main/manual-install.
Docker socket is not available. Cannot continue.
If you did this by purpose because you don't want the container to have access to the docker socket, see https://github.com/nextcloud/all-in-one/tree/main/manual-install.
Docker socket is not available. Cannot continue.
If you did this by purpose because you don't want the container to have access to the docker socket, see https://github.com/nextcloud/all-in-one/tree/main/manual-install.
Docker socket is not available. Cannot continue.
If you did this by purpose because you don't want the container to have access to the docker socket, see https://github.com/nextcloud/all-in-one/tree/main/manual-install.
Docker socket is not available. Cannot continue.
If you did this by purpose because you don't want the container to have access to the docker socket, see https://github.com/nextcloud/all-in-one/tree/main/manual-install.
Docker socket is not available. Cannot continue.
If you did this by purpose because you don't want the container to have access to the docker socket, see https://github.com/nextcloud/all-in-one/tree/main/manual-install.
Docker socket is not available. Cannot continue.

**Comment by Zoey2936** [comment, marked as answer]
can you please show your entire compose file?
